src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -128,9 +128,21 @@ internal LoggerSinkConfiguration(LoggerConfiguration loggerConfiguration, Action
             var lc = new LoggerConfiguration();
 
             _applyInheritedConfiguration(lc);
-
             configureLogger(lc);
-            return Sink(new SecondaryLoggerSink(lc.CreateLogger(), attemptDispose: true), restrictedToMinimumLevel, levelSwitch);
+
+            var subLogger = lc.CreateLogger();
+            if (subLogger.HasOverrideMap)
+            {
+                // Overrides in sub-loggers are not supported and will be ignored
```

Comment:
The comment here is probably redundant, given the content of the message we are writing to `SelfLog`.